src/PrismManager.php

```diff
@@ -81,6 +82,17 @@ protected function createOllamaProvider(array $config): Ollama
         );
     }
 
+    /**
+     * @param  array<string, string>  $config
+     */
+    protected function createMistralProvider(array $config): Mistral
+    {
+        return new Mistral(
+            apiKey: $config['api_key'] ?? '',
```

Comment:
We should make the API key required, so we can remove the default here.

Reply:
Pushed the changes. [9802f42](https://github.com/echolabsdev/prism/pull/13/commits/9802f423c1b91da29b93098baab58e8e505dd8b5)